recipe_scrapers/keukenliefdenl.py

```diff
@@ -46,7 +47,7 @@ def ingredients(self):
             )
 
         # Nothing found, we give up.
-        return []
+        raise ElementNotFoundInHtml("Could not find ingredients.")
```

Comment:
Cool.  I did wonder about using that exception type too - I think that makes sense (maybe we should update the comment on the exception class itself..) :+1: